src/transaction/TransactionController.ts

```diff
@@ -84,6 +84,15 @@ export enum TransactionStatus {
   unapproved = 'unapproved',
 }
 
+/**
+ * The device that confirmed a transaction.
+ */
+export enum ConfirmedDeviceTransaction {
```

Comment:
Nit: It's not a huge deal but I find this name... confusing :thinking:  It sounds like this is for a _device_ that is confirmed? Or a transaction that confirms a device?

Maybe something like `ConfirmedTransactionDevice` would be better. Or `WalletDevice`.

Reply:
I went with `WalletDevice`